packages/drawer/src/Drawer.tsx

```diff
@@ -49,6 +50,7 @@ const Drawer = ({
   onCancel,
   size = 'small',
   'data-test-id': testId = 'drawer',
```

Comment:
I wonder if this might be easier to reason about if we inverted the condition, e.g. a `showCancelButton` prop with a default of `true`?

I understand framing it as a deviation from the default behavior, but in general I think negative conditions are harder to parse (that's personal/subjective though).

Reply:
That's actually what I started with! However the ergonomics of `showCancel={false}` in the tags felt off to me vs a more declarative `hideCancel` - like you're conveying to the code reader what the drawer is doing.

Reply:
I admit it's weird to reason about in the code but I think it makes enough of a positive difference to the interface to offset that confusion.

Reply:
Looks like there is precedence for `hide*` style props in LaunchPad, too